### ISSUE TITLE
Tome Rater Review

### ISSUE THREAD

**Issue by franfranfranfran** [mmadavejones/pwp-capstones#1]
### Criteria 1: Valid Python Code
* _Score Level:_ 3
* _Comment(s):_ Great work creating valid and functional code for the most part. It is crucial to test all your methods as you work along. Take a closer look at the [change_email](https://github.com/mmadavejones/pwp-capstones/blob/master/Capstone%20Project/TomeRater.py#L13-L17) method. This method isn't used anywhere else so it's also possible that you could just get rid of it entirely. 
 
### Criteria 2: Implementation of Project Requirements
* _Score Level:_ 4
* _Comment(s):_ Excellent job implementing all the necessary requirements for the project! Nice work going above and beyond by adding in the equivalency methods as well. As a next step, try working on the further challenges given in the instructions like adding a new price variable.
 
### Criteria 3: Software Architecture
* _Score Level:_ 4
* _Comment(s):_ Code is separated into distinct classes and functions each which are invoked for their own purposes. There is a coherent structure to how each class builds off the others.
 
### Criteria 4: Uses Python Language Features
* _Score Level:_ 4
* _Comment(s):_ Great use of built-in python language features like format, values, items, and others throughout the program! There's a couple little areas to look at for potential tweaks - 
-You can actually leave off the keys method [here](https://github.com/mmadavejones/pwp-capstones/blob/master/Capstone%20Project/TomeRater.py#L187). It will work how you expect it to without that method.
-It'd be a bit wordy, but you could just return the value right away [here](https://github.com/mmadavejones/pwp-capstones/blob/master/Capstone%20Project/TomeRater.py#L85-L88) and avoid the extra if / else. 
 
### Overall Score: 15/16
Great work on this project! With a few key tweaks, things will be looking perfect. Nice work use the short, descriptive comments throughout as well. These kinds of things go a long way to make your code easier to read and understand. 
Something to consider moving forward, now that you have mastered a good deal of the basics, is the official [python style guide](https://www.python.org/dev/peps/pep-0008/#maximum-line-length). As an example, that link will take you to the section about suggested max line length. [Here](https://stackoverflow.com/questions/53162/how-can-i-do-a-line-break-line-continuation-in-python) is a good discussion on how to properly spill over to the next line as well.
Have fun and keep up the good work!

**Comment by mmadavejones** [comment]
Hi,

Thanks so much for taking the time to review my code and for providing the valuable feedback. This is my first time taking any of these Codecademy classes, so I'm not really sure what I'm supposed to do next. I assumed this was one shot, and I either passed or failed. Do I need to fix the issues you mentioned and resubmit, or am I done?

Thanks again!

David
